openvdb/CHANGES

```diff
@@ -15,6 +15,10 @@ Version 6.1.1 - In development
     - Fixed a bug in tools::computeScalarPotential() that could produce
       a corrupt result due to invalid memory access.
       [Reported by Edwin Braun]
+    - Fixed a bug in isInfinite() and isNan() methods of math::Tuple and
```

Comment:
This is a workaround for a bug in VS, not a bug in OpenVDB.  C++11 requires `std::isfinite` and `std::isnan` to be overloaded for integral types.

Reply:
To clarify - you're happy with the change itself but you'd like it to be recorded as a VS workaround in the release notes as opposed to a VDB bug?

Reply:
I would prefer that we deprecate functions like `math::isFinite`, `math::Sqrt`, etc. for which there are standard library equivalents, but it sounds like that's not going to be possible any time soon.

Reply:
It's still an issue in Visual Studio 2019, so no, I don't think this is going away any time soon.

I've improved the change notes.